library/psa_crypto.c

```diff
@@ -771,6 +797,31 @@ static int psa_key_policy_permits( const psa_key_policy_t *policy,
     return( 0 );
 }
 
+/** Restrict a key policy based on a constraint.
+ *
+ * \param[in,out] policy    The policy to restrict.
+ * \param[in] constraint    The policy constraint to apply.
+ *
+ * \retval #PSA_SUCCESS
+ *         \c *policy contains the intersection of the original value of
```

Comment:
I used `\c` because the argument isn't just a parameter name. The sentence is about the structure that `policy` points to, not about `policy` itself, so `\p policy` would be confusing (at least, it would confuse me, that's why I wrote `*policy`.